adcm_aio_client/core/client/_client.py

```diff
@@ -50,10 +60,32 @@ async def build_client(
     *,
     verify: Verify | None = None,  # noqa: ARG001
     cert: Cert | None = None,  # noqa: ARG001
-    timeout: float = 0.5,
+    timeout: float = 1.5,
     retries: int = 5,
     retry_interval: float = 5.0,
 ) -> ADCMClient:
+    adcm_version = await _get_and_check_adcm_version(url=url, timeout=timeout)
     requester = DefaultRequester(base_url=url, retries=retries, retry_interval=retry_interval, timeout=timeout)
     await requester.login(credentials=Credentials(username="admin", password="admin"))  # noqa: S106
-    return ADCMClient(requester=requester, bundle_retriever=BundleRetriever())
+    return ADCMClient(requester=requester, bundle_retriever=BundleRetriever(), adcm_version=adcm_version)
+
+
+async def _get_and_check_adcm_version(url: str, timeout: float) -> str:
+    try:
+        adcm_version = await _get_adcm_version(url=url, timeout=timeout)
+    except VersionRetrievalError as e:
+        message = f"Can't get ADCM version for {url}"
```

Comment:
let's specify that it's more likely due to adcm ver < 2.5.0

Reply:
Added line to error